Readme.md

```diff
@@ -111,26 +111,28 @@ If you already have Python and NodeJS installed, you can skip to step 3.
 4. Right click the zip and open Properties > General tab and check _Unblock_ if it appears at the bottom. Then Unzip it.
 5. Run powershell as administrator, [set permissions](https://lazyadmin.nl/powershell/running-scripts-is-disabled-on-this-system/)
 ```shell
-# only need to set this one time in powershell
+# setting this one time in powershell normally persists on reboots.
 Set-ExecutionPolicy RemoteSigned
 ```
-### Run
-2. Change directory, `cd` to the location of the PineSAM folder that was unzipped above.
+#### Run
+1. Change directory, `cd` to the location of the PineSAM folder that was unzipped above.
 ```shell
-setup-dev.bat   # only need to run this one time
+setup-dev.bat   # only need to run this one time for each new version
 run-dev.bat     # run this command every time you use Pinecil
 ```
+2. See [Known Issues](https://github.com/builder555/PineSAM/Readme.md#known-issues) below or the [Discussions](https://github.com/builder555/PineSAM/discussions) if you run into problems.
```

Comment:
probably redundant also

Reply:
I moved discussions to  known issues.
since  people might only zoom in to  Linux or Mac or  Windows  install sections, I put a sentence to go to Known  issues if  it doesn't work.  per Distro section  since a Windows person is not going to look at the linux instructions to find that. 
take a look at it now or do you think  sentence to go to known issues should be removed from all 3 category sections?